inst/examples/hierarchical_linear_slopes_corr.Rmd

```diff
@@ -0,0 +1,32 @@
+### Random intercept-slope model (with correlated effects)
+
+A hierarchical, Bayesian linear regression model using the iris data, with random intercepts and slopes for each of the three species.
+The slopes and intercepts for each species are *correlated* in this example.
+
+```{r hierarchical_linear_slopes_corr_greta}
+# model matrix
+modmat <- model.matrix(~ Sepal.Width, iris) 
+# index of species
+jj <- as.numeric(iris$Species)
+
+M <- ncol(modmat) # number of varying coefficients
+N <- max(jj) # number of species
+
+# define the varying coefficient
+tau <- exponential(0.5, dim = M)
+Omega <- lkj_correlation(3, M)
```

Comment:
Could you please add a comment about here saying that the following code is a more computationally efficient and stable way of sampling `ab` as a multivariate normal, with covariance Sigma